samples/basic/blinky/src/main.c

```diff
@@ -46,6 +46,8 @@ void main(void)
 	while (1) {
 		gpio_pin_set(dev, PIN, (int)led_is_on);
 		led_is_on = !led_is_on;
+		printk("Going to sleep..\n");
```

Comment:
We should not be editing the sample apps at all. This will never get accepted upsteream.